examples/exchange_client/insurance_rpc/1_InsuranceFunds.py

```diff
@@ -5,7 +5,8 @@
 from pyinjective.constant import Network
 
 async def main() -> None:
-    network = Network.testnet()
+    # select network: local, testnet, mainnet
+    network = Network.mainnet()
```

Comment:
All examples should reflect testnet.

Reply:
Testnet has no InsuranceFunds currently—example will return no results